src/fu-util.c

```diff
@@ -977,6 +978,40 @@ fu_util_device_test_filename(FuUtilPrivate *priv,
 	return TRUE;
 }
 
+static gboolean
+fu_util_inhibit(FuUtilPrivate *priv, gchar **values, GError **error)
+{
+	const gchar *reason = "not set";
+	g_autofree gchar *inhibit_id = NULL;
+
+	if (g_strv_length(values) > 0)
+		reason = values[1];
+
+	/* inhibit then wait */
+	inhibit_id = fwupd_client_inhibit(priv->client, reason, priv->cancellable, error);
+	if (inhibit_id == NULL)
+		return FALSE;
+	g_print("Inhibit ID is: %s, use ctrl-c to abort\n", inhibit_id);
```

Comment:
presumably this should be translatable

Reply:
How about this:

![Screenshot from 2023-01-23 09-32-17](https://user-images.githubusercontent.com/151380/214006711-4fe71469-9eca-4f92-bbd3-4a74da4131a1.png)

Reply:
Lgtm